docs/user/rstudio/ide/guide/tools/terminal.qmd

```diff
@@ -2,93 +2,100 @@
 
 ## Execute in Terminal
 
-When editing files in the RStudio editor, any selection (or the current line if nothing is selected) can be sent to the active terminal via **Ctrl+Alt+Enter** (also **Cmd+Alt+Enter** on the Mac). If a single-line was sent to the terminal the cursor will advance automatically to the next line, allowing single-stepping.
+When editing files in the RStudio editor, any selection (or the current line if nothing is selected) can be sent to the active terminal via <kbd>Ctrl+Alt+Enter</kbd> (<kbd>Cmd+Alt+Enter</kbd> on the Mac). If a single-line was sent to the terminal the cursor will advance automatically to the next line, allowing single-stepping.
+
+In following example:  
 
-In following example, a simple Python source file was loaded, a Python interpreter started in the current terminal, and Ctrl+Alt+Enter used to step through the script:
+1. A simple Python source file was loaded
+2. A Python interpreter started in the current terminal
+3. Ctrl+Alt+Enter used to step through the script:
 
-![](images/terminal-python.png){fig-alt="screenshot of a simple Python repl example"}
+![](images/terminal-python.png){fig-alt="screenshot of a simple Python REPL example"}
 
 This is just an example; for more powerful integrated Python support [look at the reticulate package](https://rstudio.github.io/reticulate/).
 
-If a shell file is being edited (*\*.sh*), **Ctrl+Enter** (plus **Cmd+Enter** on Mac) can be used to single-step, as can the **Run** toolbar button. This will also work on Windows, but no check is done that the shell-type loaded in the active terminal session matches the commands being sent.
+If a shell file is being edited (`*.sh`), <kbd>Ctrl+Enter</kbd> (<kbd>Cmd+Enter</kbd> on Mac) can be used to single-step, as can the **Run** toolbar button. This will also work on Windows, but no check is done that the shell-type loaded in the active terminal session matches the commands being sent.
+
 
-\
 ![](images/terminal-shell.png){fig-alt="screenshot example of running a shell script"} 
 
-Note that the **Run Script** button does the same as prior versions of RStudio, and will save and run the entire script via **system()** in the R Console. This button does not appear in Windows.
+Note that the **Run Script** button does the same as prior versions of RStudio, and will save and run the entire script via `system()` in the R Console. This button does not appear in Windows.
 
 ## Busy Terminals
 
 Programs running in a terminal do not block the rest of the RStudio user-interface, so work can continue working in RStudio even when one or more terminal sessions have a child process running.
 
-A terminal with child processes is considered *busy* by RStudio, and will have **(busy)** next to its name, and the close \[x\] changes to a **stop** button.
+A terminal with child processes is considered *busy* by RStudio, and will have:
+
+1. **(busy)** next to the terminal name.
+2. The close **x** changes to a **STOP** button.
```

Comment:
This could also be an unordered list

Reply:
I refer to these in the screenshot by number. Is that ok?

<img width="1073" alt="terminal-sleep" src="https://user-images.githubusercontent.com/29187501/193869191-4c377375-1e01-4655-8944-3effd5566384.png">

Reply:
@AshleyHenry15 - does that work ok?